person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_accessor :id, :name, :age, :parent_permission
+
+  def initialize(id, name, age, parent_permission: true)
```

Comment:
- [ ] Consider adding optional value for name attribute to be like this `name = "Unknown"`